.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to the Salesforce CLI
+Welcome, and thank you for your interest in contributing to the Salesforce CLI!
+
+There are multiple ways you can contribute:
+
+### Report an issue
+Did you find a bug? First, make sure the bug wasn't already reported by searching on the `issues` section: https://github.com/forcedotcom/cli/issues.
+If you don't find an issue, then open a new one and follow the bug template. Minimal repros are highly encouraged as these help us to quickly identify the root cause of the issue.
+
+### Pull requests
+We accept pull requests for bug fixes and feature requests for any issues labeled with [`help wanted`](https://github.com/forcedotcom/cli/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
```

Comment:
Why only issues with `help wanted`? I don't want to discourage folks from not contributing to other issues.

Reply:
Both `help wanted` and `good first issue` labels are used in OSS projects to mark them as open for contributions.

Sometimes we get feature requests/bug fixes that, while easy to add/fix, might require more context/design decision from us before starting any work. See this one for instance: https://github.com/forcedotcom/cli/issues/1790

What I would like to avoid is contributors to work on something and then having us closing their PRs because we decide not to add a feature/take another approach.

When triaging issues we take the decision whether to accept 3rd party contributions or not.
Maybe we should start using more labels so that it's clear a GH issue will be assigned to us and not open to contributions, like `core`.